src/Stratis.Bitcoin.Features.Miner/Stratis.Bitcoin.Features.Miner.csproj

```diff
@@ -23,6 +23,10 @@
     <EmbeddedResource Remove="Stratis.Bitcoin.Features.Miner\**" />
     <None Remove="Stratis.Bitcoin.Features.Miner\**" />
   </ItemGroup>
+
+  <ItemGroup>
```

Comment:
This change is probably not needed since AssemblerFactory was removed.

Reply:
Removed